<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>32</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Conexão</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="Desconectar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Desconectar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="HostLabel">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Host</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="HostInput">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>100</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>192.168.0.100</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10.0.0.119</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PortaLabel">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Porta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PortaInput">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>4000</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Conectar">
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cadastro de palavras</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>A</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>D</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>E</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>F</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>G</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>H</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>I</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>J</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>K</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>L</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Q</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>R</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>T</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>U</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>V</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>W</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Z</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Cadastrar">
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Utilização da luva</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="stop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>1000000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Config</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
